### ISSUE TITLE
Why require project Id?

### ISSUE THREAD

**Issue by BearOutThere** [jonah-katz/cheetah-track#1]
Just setup Cheetah in our slack tool and tried to start task tracking but it appears that it's expecting Toggl's project id which is an 8 digit code.  I can't imagine anybody would bother memorizing these.  It would be much better to specify projects by their names.

**Comment by vzqzac** [comment]
As I agree with this issue, it's also true that you can run '/cheetah projects', that will display your toggl projects with the associated project id, not a very friendly solution but you won't have to memorize them at least until new features are added

**Comment by BearOutThere** [comment]
Toggl browser plugin makes this a moot point since new tasks can be started and assigned to proper projects without the id.  The very fact that this basic usability issue is still not addressed indicates that it's not a priority for the Cheetah team.  Waiting "until new features are added" might be a long time indeed.

**Comment by vzqzac** [comment]
Indeed, even the 6-months-no-response for the issue implicated it, really sad as there are no many Toggl integrations for slack.

**Comment by jonah-katz** [comment]
@bearoutthere Wow, I can't believe I missed this issue for so long. At least its been only 3 days more than a year 🙈. Anyway, as @VzqzAc said, the goals of `/cheetah projects` is to make it quick and easy to copy and paste. The reason we don't do project names is it'd be hard to then add more params to the command. Perhaps a multi-message process would be better:

1. /cheetah
2. Which project? (1) Project 1 (2) Project 2 ..
3. 1
4. Timer running!

Thoughts?
